cpp/src/io/utilities/block_utils.cuh

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <stdint.h>
+
+namespace cudf {
+namespace io {
+
+#if (__CUDACC_VER_MAJOR__ >= 9)
```

Comment:
Since our minimum CUDA version is 9.2 anyways, why not get rid of the macros all together?

Reply:
I actually still do some development on Windows with cuda8 on Maxwell, where I use the same source files :)

Reply:
Okay, let's keep this then until you move your Windows environment to CUDA 9.2 or later.